web/src/features/fba/components/AboutDataPopover.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react'
+import Popover from '@mui/material/Popover'
+import Typography from '@mui/material/Typography'
+import InfoIcon from '@mui/icons-material/Info'
+import { theme } from 'app/theme'
+
+interface AboutDataProps {
+  advisoryThreshold: number
+}
+
+const AboutDataPopover = ({ advisoryThreshold }: AboutDataProps) => {
+  const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null)
+
+  const handlePopoverOpen = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget)
+  }
+
+  const handlePopoverClose = () => {
+    setAnchorEl(null)
+  }
+
+  const open = Boolean(anchorEl)
+
+  return (
+    <div data-testid="about-data-popover">
+      <Typography
+        data-testid="about-data-trigger"
+        fontSize={'0.75rem'}
+        onClick={handlePopoverOpen}
+        sx={{ cursor: 'pointer', display: 'flex', alignItems: 'center' }}
+      >
+        <InfoIcon sx={{ marginRight: theme.spacing(1), color: '#4681f4' }}></InfoIcon>
+        About this data
+      </Typography>
+      <Popover
+        id="about-data-popover"
+        open={open}
+        anchorEl={anchorEl}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'left'
+        }}
+        onClose={handlePopoverClose}
+        disableRestoreFocus
+        slotProps={{ paper: { sx: { maxWidth: 350, backgroundColor: '#EEEEEE' } } }}
+      >
+        <Typography
+          sx={{
+            padding: theme.spacing(1),
+            fontWeight: 'bold',
+            fontSize: '1rem'
+          }}
+        >
+          About This Data
+        </Typography>
+        <div
+          style={{ paddingBottom: theme.spacing(3), paddingRight: theme.spacing(2), fontSize: '0.9rem' }}
+          data-testid="about-data-content"
+        >
+          <ul style={{ margin: 0 }}>
+            <li>
```

Comment:
The markdown for this list is broken in the deployment for some reason and I only see a single bullet point in the list.
![image](https://github.com/user-attachments/assets/f2bad8f4-cbb2-4a40-9eb1-ae2c147b0962)

Reply:
![image](https://github.com/user-attachments/assets/8527fdda-563a-41fd-88d9-8578511934b0)

Reply:
Might need to make the entire string interpolated to get around it.

Reply:
Huh, that's interesting. I see all the bullets locally

Reply:
Looks like just re-deploying took care of it....odd

Reply:
Very strange!